sync.go

```diff
@@ -23,21 +20,14 @@ func (s spec) sync() {
 			fatal("failed to ensure tool dir", err)
 		}
 
-		// Download everything to cache
+		// Download everything to tool directory
 		for _, t := range s.Tools {
 			err = download(t)
 			if err != nil {
 				fatalExec("failed to sync "+t.Repository, err)
```

Comment:
Not a comment on this PR, but a general one, this kind of behavior can end up with incomplete syncs.

Might want to consider downloading into a `/tmp` folder, and copying over at the end. not sure